azure-pipelines.yml

```diff
@@ -0,0 +1,65 @@
+variables:
+  app_id: 'game.shader.presets'
+
+trigger:
+  branches:
+    include:
+    - Matrix
+    - releases/*
+  paths:
+    include:
+    - '*'
+    exclude:
+    - 'debian/*'
+
+jobs:
+  - job: Windows
+
+    pool:
+      vmImage: 'VS2017-Win2016'
+
+    strategy:
+      matrix:
+        Win32:
+          GENERATOR: "Visual Studio 15 2017"
+          ARCHITECTURE: Win32
+          CONFIGURATION: Release
+        Win64:
+          GENERATOR: "Visual Studio 15 2017"
+          ARCHITECTURE: x64
+          CONFIGURATION: Release
+        Win64-UWP:
+          GENERATOR: "Visual Studio 15 2017"
+          ARCHITECTURE: x64
+          CONFIGURATION: Release
+          WINSTORE: -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION="10.0.17763.0"
+        ARM64-UWP:
+          GENERATOR: "Visual Studio 15 2017"
+          ARCHITECTURE: ARM64
+          CONFIGURATION: Release
+          WINSTORE: -DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION="10.0.17763.0"
+
+    workspace:
+      clean: all
+
+    steps:
+
+    - script: |
+        cd ..
+        git clone --branch master --depth=1 https://github.com/xbmc/xbmc.git kodi
```

Comment:
We'll need to clone a retroplayer branch to pull in the Shader Preset API. I'll push a branch named `retroplayer-19alpha1` to https://github.com/kodi-game/xbmc.git

Reply:
Pushed: https://github.com/kodi-game/xbmc/tree/retroplayer-19alpha1